client/src/components/projects/project-card/ProjectCard.test.ts

```diff
@@ -0,0 +1,28 @@
+import Projects from '../Projects.vue';
+import ProjectCard from './ProjectCard.vue';
+import store from '../../../store/index';
+import {mount} from '@vue/test-utils';
+
+describe('Project Card', () => {
+
+  const projects = mount(Projects, {store});
```

Comment:
No. Unit test only tests one **Unit** e.g. ProjectCard in this case - mock object instead of using parent